src/middlewared/middlewared/etc_files/local/ssh/sshd_config.mako

```diff
@@ -48,6 +48,12 @@
 	users = middleware.call_sync('user.query')
 	root_user = filter_list(users, [['username', '=', 'root']], {'get': True})
 
+	login_banner = middleware.call_sync('system.advanced.login_banner')
+	if login_banner != '':
+		with open('/etc/login_banner', 'w', encoding='utf-8') as f:
```

Comment:
I think we should remove this file when `login_banner` is empty string ?

Reply:
What's the preferred way to delete a file? Pathlib unlink, or os.remove after checking for it existing?

Reply:
You can do any i guess and no need to check first if the file exists or not as that can potentially lead to TOCTOU issue, instead you can catch file not found exception with `contextlib.suppress` or try/catch

Reply:
Gotcha, I'll use os just because it's already being imported